site/content/installation/installing-ngf/manifests.md

```diff
@@ -138,6 +138,16 @@ kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric
 
 {{% /tab %}}
 
+{{%tab name="NGINX in debug mode"%}}
```

Comment:
@ADubhlaoich do you think it's a good idea to include an example for how to install NGF in debug mode in our installation docs?

Reply:
I think including an example makes sense, though if it's an uncommon use case, it might make more sense to have it in documentation specific to debugging.

Is debug mode used by developers or users more often?

(Developer being someone working on NGF itself).

Reply:
Not sure if @kate-osborn or @sjberman have any input on this, but from what I remember Kate telling me -- it would be a bad idea for users to run with nginx in debug mode in prod environments.

I personally haven't needed to run with NGINX in debug mode, and I don't see a user's need either (as we provide so many other tools/logs/errors for debugging issues). So I think it would be an uncommon use case. 

I think it would be beneficial to move it to documentation specific to debugging, @ADubhlaoich in that case would you agree?

Reply:
> Is debug mode used by developers or users more often?

I think the common use case will be support. The support team may need additional nginx logs in order to diagnose an issue.

Reply:
Chose to remove the full example of a Kubernetes Manifest for running nginx in debug mode. Also means removing this section. 

I think that the given instruction for running NGINX in debug mode in the `data-plane-configuration.md` is sufficient for anyone at the level of looking at NGINX debug logs. Also I think most people would use Helm anyways which goes around this issue entirely. By not including the entire nginx-debug Kubernetes Manifest I think it helps not overcrowd with extra information -- and like @sjberman commented below -- would not allow for the risk of 

> ...someone may just see it and try to install it but it will fail since they aren't using the debug image.

Willing to change though based on others thoughts!